packages/contracts/.addresses.json

```diff
@@ -1,10 +1,14 @@
 {
     "11155111": {
-		"template": "0xc8000f0CC97e5661DCC096a2dE5aCf5f55F04f29",
-        "arbitrator": "0x96073897873796d1950B1B04Fe2Ead8E0CA34914"
+		"template": "0x2F8e3C513aB9e75D9E2B8F4C683E2577c908a411",
```

Comment:
Did we miss the formatting here?

Reply:
Whoops, I think tabs went through instead of spaces. Addresses in b86dfe2695550b9c73bb366a6cbac1fbc50f6657